qiskit/circuit/quantumcircuit.py

```diff
@@ -1217,6 +1217,9 @@ def draw(self, output=None, scale=None, filename=None, style=None,
         below:
 
         Args:
+            name (str): The name of the style. The name can be set to 'iqx',
+                'bw', or 'default'. This overrides the setting in the
+                '~/.qiskit/settings.conf' file.
```

Comment:
I think this is redundant with `style={'name':'iqx'})`. Probably the documentation on `style` should be extended. with possible `names`.

Reply:
`name` was already in the style dict but not accessible by the user. We made it changeable by the user through the style dict and added it to the docs, and we now use it as a way to select color style.